templates/tower-project.yaml

```diff
@@ -225,7 +225,7 @@ Resources:
   TowerBucket:
     Type: AWS::S3::Bucket
     Properties:
-      BucketName: !Sub "${AWS::StackName}-tower-bucket"
+      BucketName: !Sub "${AWS::StackName}-tower-bucket" #would it make sense to append project bucket names with a tenant name?
```

Comment:
These buckets might have multiple tenants (assuming you're referring to users who will be uploading and downloading to the bucket). Are you thinking of something else?

Reply:
I was just thinking of the naming convention getting out of hand... if everything has a static prefix it would be simpler to find them later.